public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -368,168 +471,222 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
+                  queries={queries}
+                  panelData={queryPreviewData}
+                  condition={condition}
+                  onSetCondition={handleSetCondition}
+                  onRemoveExpression={(refId) => {
+                    dispatch(removeExpression(refId));
+                  }}
+                  onUpdateRefId={onUpdateRefId}
+                  onUpdateExpressionType={(refId, type) => {
+                    dispatch(updateExpressionType({ refId, type }));
+                  }}
+                  onUpdateQueryExpression={(model) => {
+                    dispatch(updateExpression(model));
+                  }}
+                />
+              </>
+            )}
+            {/* action buttons */}
+            <Stack direction="column">
+              {!isAdvancedMode && (
+                <SimpleConditionEditor
+                  simpleCondition={simpleCondition}
+                  onChange={setSimpleCondition}
+                  expressionQueriesList={expressionQueries}
+                  dispatch={dispatch}
+                  previewData={queryPreviewData[condition ?? '']}
+                />
+              )}
+              <Stack direction="row">
+                {isAdvancedMode && config.expressionsEnabled && <TypeSelectorButton onClickType={onClickType} />}
+
+                {isPreviewLoading && (
+                  <Button icon="spinner" type="button" variant="destructive" onClick={cancelQueries}>
+                    Cancel
+                  </Button>
+                )}
+                {!isPreviewLoading && (
+                  <Button
+                    data-testid={selectors.components.AlertRules.previewButton}
+                    icon="sync"
+                    type="button"
+                    onClick={() => runQueriesPreview()}
+                    disabled={emptyQueries}
+                  >
+                    Preview
+                  </Button>
+                )}
+              </Stack>
+            </Stack>
+
+            {/* No Queries */}
+            {emptyQueries && (
+              <Alert title="No queries or expressions have been configured" severity="warning">
+                Create at least one query or expression to be alerted on
+              </Alert>
             )}
           </Stack>
-
-          {/* No Queries */}
-          {emptyQueries && (
-            <Alert title="No queries or expressions have been configured" severity="warning">
-              Create at least one query or expression to be alerted on
-            </Alert>
-          )}
-        </Stack>
-      )}
-    </RuleEditorSection>
+        )}
+      </RuleEditorSection>
+
+      <ConfirmModal
+        isOpen={showResetModeModal}
+        title="Switching to simple mode"
+        body="Is not possible to convert the existing query and expressions to simple mode. 
+          If you continue, the current queries and expressions will be lost. Are you sure you want to switch to simple mode?"
```

Comment:
*It is not possible

@brendamuir Do you have any thoughts on the overall descriptions here? (screenshot for your reference)

![image](https://github.com/user-attachments/assets/d16cd1f4-f677-46d8-97f9-1b723ceff6ac)

Reply:
how about this: title: Switch to simple mode. text: The selected queries and expressions cannot be converted to simple mode. Switching will remove them. Do you want to proceed?

Reply:
👌